### DISCUSSION TITLE
behoefte aan preview in CMS voor PDC & OpenPub-pagina’s

### DISCUSSION THREAD

**Discussion by PimSV** [OpenWebconcept/Afspraken#17] [Q&A] [answered]
OWC-community,

Wij hebben binnen Stichtse Vecht echt behoefte aan een preview bij PDC & OpenPub-pagina’s in Wordpress voordat je een pagina live zet/publiceert. Je wil zien hoe iets eruit ziet voordat je het live zet. Ik zie dit als een standaardfunctionaliteit van een CMS, maar zit er niet in.

Heeft 1 van jullie hier een oplossing/workaround voor (of zijn we de enige met deze behoefte)? 
Dank voor het meedenken!

**Comment by sanderdekroon** [comment, marked as answer]
Hi Pim! Dit is al mogelijk als je op een PDC/OpenPub endpoint `?draft-preview=true` meegeeft. Bijvoorbeeld: `https://pdc.website.nl/wp-json/owc/pdc/v1/themas/ongepubliceerde-thema?draft-preview=true`.

Afhankelijk van hoe de 'voorkant' geïmplementeerd is, kun je dan een item bekijken via: `https://website.nl/thema/ongepubliceerde-thema?draft-preview=true`. Althans, dat is ik/wij het geïmplementeerd heb(ben) :).

**Reply by PimSV** [reply]
Dank Sander, dat gaan we ook proberen!
